Device.xcodeproj/project.pbxproj

```diff
@@ -436,6 +436,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE = "";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				TARGETED_DEVICE_FAMILY = "1,2";
```

Comment:
In order for this project to run on the iPad, you need to allow the iPad device target.

| Description: | Comma-separated list of numeric identifiers. Specifies the device families on which the product must be capable of running. |
|:---|:---|
| Identifiers: | 1: iPhone/iPod touch.  2: iPad. |
|Default value: | 1 |
| Example value: | 1,2 |

Source: https://developer.apple.com/library/mac/documentation/DeveloperTools/Reference/XcodeBuildSettingRef/1-Build_Setting_Reference/build_setting_ref.html#//apple_ref/doc/uid/TP40003931-CH3-SW165